.ci/integration/docker-compose.override.yml

```diff
@@ -18,4 +18,4 @@ services:
     image: rabbitmq:latest
     hostname: rabbit
     ports:
-      - "5672:5672"
+      - "15672:5672"
```

Comment:
5672 not available on travis runners it looks like. Will have to do some digging as to why and clean up this PR, but previously docker-compose was failing outright not being able to bind that port.

Reply:
good find ,thanks for addressing this.